package-res/ccc/core/base/axis/size-axis.js

```diff
@@ -36,7 +36,15 @@ def('pvc.visual.SizeAxis', pvc_Axis.extend({
             if(def.debug >= 4) def.log("Scale: " + def.describe(def.copyOwn(scale)));
 
             return this;
-        }
+        },
+
+        // CDF603 
+        /* Specify a default FixedLength 
+           Eg. used when imposing ratio through sliding window */
+        setInitialLength: function(fixedLength){
+            this.option.defaults({ 'FixedLength': fixedLength });
```

Comment:
Please do not put initial and final spaces in object literals: `{'FixedLength': fixedLength}`.